### ISSUE TITLE
Cannot use e "__Schema" from another module or realm

### ISSUE THREAD

**Issue by rjjlucy** [aws-amplify/amplify-js#4194]
**Describe the bug**
Observe the error in developer console in Chrome when I run the app locally.
```
Uncaught Error: Cannot use e "__Schema" from another module or realm.

Ensure that there is only one instance of "graphql" in the node_modules
directory. If different versions of "graphql" are the dependencies of other
relied on modules, use "resolutions" to ensure only one version is installed.

https://yarnpkg.com/en/docs/selective-version-resolutions

Duplicate "graphql" modules cannot be used at the same time since different
versions may have different capabilities and behavior. The data from one
version used in the function from another could produce confusing and
spurious results.
    at exports.default (2c943a20-28d2-4949-8560-ff3f98870d02:182)
    at s (2c943a20-28d2-4949-8560-ff3f98870d02:182)
    at i (2c943a20-28d2-4949-8560-ff3f98870d02:182)
    at D (2c943a20-28d2-4949-8560-ff3f98870d02:182)
    at q (2c943a20-28d2-4949-8560-ff3f98870d02:182)
    at new O (2c943a20-28d2-4949-8560-ff3f98870d02:182)
    at O (2c943a20-28d2-4949-8560-ff3f98870d02:182)
    at Object../node_modules/graphql/type/introspection.js (2c943a20-28d2-4949-8560-ff3f98870d02:182)
    at t (2c943a20-28d2-4949-8560-ff3f98870d02:1)
    at Object../node_modules/graphql/type/validate.js (2c943a20-28d2-4949-8560-ff3f98870d02:182)
```

**To Reproduce**
1. Integrate with `Amplify` and import it like this
`import Amplify, { Analytics } from 'aws-amplify';`
2. Regular way to consume it
`Amplify.configure(amplifyConfig);`
`Analytics.record(event);`
 
Version: Amplify 1.1.40

**Comment by rjjlucy** [comment]
I tried to set `process.env.NODE_ENV = 'production'` and it seems like the error is gone. But it is not an ideal solution for us. We cannot simply set it to `production` when it's actually `development`

**Comment by Amplifiyer** [comment]
@rjjlucy, have you tried resolving the graphql version to 0.13 in your package.json like below:
```
"resolutions": {
    "aws-amplify/**/graphql": "^0.13.0"
}
```

**Comment by ehemmerlin** [comment]
@Amplifiyer I resolved the graphql version to 0.13.0 using the code you provided but fixing this error thows another one (Uncaught ReferenceError: process is not defined). A solution is to upgrade GraphQL from 0.13.0 to 14.0.0 unfortunatly GraphQL 0.13.0 is an AWS Amplify API dependency. See the issue I reported yesterday here #4217
